src/api/utils/userAgent.ts

```diff
@@ -0,0 +1,26 @@
+import bowser from "bowser";
+import { isBrowser, isNode } from "browser-or-node";
+
+const CLIENT_NAME = "univapay-node";
+
+function getPlatform() {
+    if (isNode) {
+       return `node.js@${process.versions.node}`;
+    }
+
+    if (isBrowser) {
+        const { browser: { name, version } } = bowser.parse(window.navigator.userAgent);
+
+        return `${name}@${version}`;
+    }
+
+    return "uknown-platform"
+}
+
+function getModuleType() {
```

Comment:
Normally we try to go for arrow function when we do not need to bind. But the nodes being very weird feel free to ignore this coment.